flowplayer.hlsjs.js

```diff
@@ -602,7 +602,20 @@
                                             maxLevel = hls.loadLevel;
                                         }
                                         break;
-
+                                    case "FRAG_PARSING_METADATA":
+                                        data.samples.forEach(function(sample) {
+                                          var metadataHandler = function() {
+                                            if (videoTag.currentTime < sample.dts) return;
+                                            videoTag.removeEventListener('timeupdate', metadataHandler);
+                                            var raw = new TextDecoder('utf-8').decode(sample.data);
```

Comment:
Is there some other way to decide that uintarray to utf8? The problem here is IE it seems.